CorsixTH/corsix-th.6

```diff
@@ -0,0 +1,84 @@
+.\" Copyright (c) 2018 Stephen "TheCycoONE" Baker
+.\"
+.\" Permission is hereby granted, free of charge, to any person obtaining a copy of
+.\" this software and associated documentation files (the "Software"), to deal in
+.\" the Software without restriction, including without limitation the rights to
+.\" use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+.\" of the Software, and to permit persons to whom the Software is furnished to do
+.\" so, subject to the following conditions:
+.\" 
+.\" The above copyright notice and this permission notice shall be included in all
+.\" copies or substantial portions of the Software.
+.\" 
+.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+.\" AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.\" LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+.\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+.\" SOFTWARE.
+.\"
+.\" This manpage is written in mdoc(7).
+.\" Language reference: https://man.openbsd.org/mdoc.7
+.Dd August 22, 2018
+.Dt CORSIX-TH 6
+.Os
+.Sh NAME
+.Nm corsix-th
+.Nd "An" open source implementation "of" Theme Hospital
+.Sh SYNOPSIS
+.Nm
+.Op Fl -bitmap-dir Ns Ar =path
+.Op Fl -config-file Ns = Ns Ar path
+.Op Fl -connect-lua-dbgp
+.Op Fl -dump=strings
+.Op Fl -interpreter Ns = Ns Ar path
+.Op Fl -load Ns = Ns Ar savegame
+.Op Fl -lua-dir Ns = Ns Ar path
+.Sh DESCRIPTION
+CorsixTH is a reimplementation of the 1997 Bullfrog business simulation game Theme Hospital.
+As well as faithfully recreating the original, CorsixTH adds support for modern operating systems, high resolution displays, and much more.
+A purchased copy of Theme Hospital, or the Theme Hospital demo is required to use CorsixTH.
```

Comment:
Isn't the demo useful for only a few levels? It now looks asif it's a full alternative for the purchased copy.
Would be fair to warn about that, I think.